src/Schema/OracleSchemaManager.php

```diff
@@ -265,11 +265,9 @@ public function createDatabase(string $database): void
     }
 
     /**
-     * @internal The method should be only used from within the OracleSchemaManager class hierarchy.
-     *
      * @throws Exception
      */
-    public function dropAutoincrement(string $table): bool
+    protected function dropAutoincrement(string $table): bool
```

Comment:
Should it also be made final or is it meant to be overridden ?

Reply:
I’d leave it as is for now. Making it final is out of scope, and there’s no problem if someone has it overridden. The issue is about public methods that require runtime assertions.